BenchmarkVCFs/SimpleBenchmark.wdl

```diff
@@ -0,0 +1,592 @@
+        File combined_IDD = CombineSummaries.IDD_combined_summaries
+        File combined_ST = CombineSummaries.ST_combined_summaries
+        File combined_ROC = CombineSummaries.ROC_combined_summaries
+    }
+
+}
+
+task SubsetVCF {
+    input {
+        File input_vcf
+        File input_vcf_index
+        String? input_sample_name
+        Reference reference
+
+        Stratifier stratifier
+        Int interval_padding = 0    # Amount of bases to add around stratification intervals
+
+        String gatk_tag
+        Int? preemptible
+        Int disk_size = 10 + ceil(4.2 * size(input_vcf, "GB") + 2.2 * size(input_vcf_index, "GB") + size(reference.fasta, "GB"))
+        Int cpu = 4
+        Int memory = 16
+    }
+
+    command <<<
+        set -xeuo pipefail
+
+        # Create symlink for VCF & index in case their paths are different, e.g. when using TDR
+        ln -s ~{input_vcf} input.vcf.gz
+        ln -s ~{input_vcf_index} input.vcf.gz.tbi
+
+        # Subset to given sample / loci; Output subsetted VCF
+        # Add variable padding to ensure capture of variants on boundaries downstream
+        # Remove unused alternates to make bcftools stats more accurate on multisample callsets
+        gatk SelectVariants \
+            -V input.vcf.gz \
+            -R ~{reference.fasta} \
+            ~{"-sn " + input_sample_name} \
+            ~{"-L " + stratifier.interval} \
+            ~{"--interval-padding " + interval_padding} \
+            --remove-unused-alternates \
+            -O subset.vcf.gz
+
+    >>>
+
+    runtime{
+        docker: "us.gcr.io/broad-gatk/gatk:" + gatk_tag
+        preemptible: select_first([preemptible, 0])
+        disks: "local-disk " + disk_size + " HDD"
+        bootDiskSizeGb: "16"
+        cpu: cpu
+        memory: memory + " GB"
+    }
+
+    output {
+        File output_vcf = "subset.vcf.gz"
+        File output_vcf_index = "subset.vcf.gz.tbi"
+    }
+}
+
+task VCFEval {
+    input {
+        # Input VCF Files
+        File call_vcf
+        File call_vcf_index
+        String call_output_sample_name
+        File base_vcf
+        File base_vcf_index
+        String base_output_sample_name
+
+        Reference reference
+
+        # Interval File to Subset Analysis on given actual truth data
+        File? evaluation_bed
+
+        # String for VCF field to use as ROC score
+        String score_field
+
+        # Label to add to ROC table outputs
+        String strat_label
+
+        # vcfeval Arguments
+        Boolean passing_only = true
+        Boolean require_matching_genotypes = true
+        Boolean enable_ref_overlap = false
+
+        # Runtime params
+        Int? preemptible
+        Int disk_size = ceil(size(call_vcf, "GB") + size(base_vcf, "GB") + size(reference.fasta, "GB")) + 25
+        Int cpu = 8
+        Int memory = 16
+    }
+
+    command <<<
+        set -xeuo pipefail
+
+        rtg format -o rtg_ref ~{reference.fasta}
```

Comment:
This took about 50 seconds to run locally. I agree it's wasteful since it'll basically be the same computation for every run, but I'm not sure of a good way to import the relevant precomputed files since it's a full directory rather than one output. I think it's probably fine for now since it's relatively fast?

Reply:
Yeah if it's 50 seconds then I guess it's pretty fast. One way to import the directory would be to make a `.tar.gz` archive out of it and extract that